modules/component-store/spec/types/component-store.types.spec.ts

```diff
@@ -25,7 +25,6 @@ describe('ComponentStore types', () => {
     describe('infers Subscription', () => {
       it('when argument type is specified and a variable with corresponding type is passed', () => {
         const effectTest = `const sub = componentStore.effect((e: Observable<string>) => number$)('string');`;
-        expectSnippet(effectTest).toSucceed();
```

Comment:
Why do we have to remove `.toSucceed()` checks? They're important because sometimes inference checks can succeed even if compilation fails.